### DISCUSSION TITLE
Просмотр локальных видео

### DISCUSSION THREAD

**Discussion by ghost** [ilyhalight/voice-over-translation#432] [Q&A] [answered]
Есть ли возможность переводить видео расположенные локально на пк, но открытые через браузер?

**Comment by NullVerdict** [comment, marked as answer]
> Есть ли возможность переводить видео расположенные локально на пк, но открытые через браузер?

Нет #310

**Comment by ghost** [comment]
А извиняюсь, тупанул, не нашёл. Спасибо за ответ.
